### ISSUE TITLE
Web and annotation file are inconsistent

### ISSUE THREAD

**Issue by chen136235** [obophenotype/human-phenotype-ontology#4198]
Disease OMIM:203200 and gene MC1R(4157) are associated at https://hpo.jax.org/app/browse/gene/4157

Why can't find this association in diseases_to_genes.txt in http://compbio.charite.de/jenkins/job/hpo.annotations.monthly/lastStableBuild/

There are many such diseases and genes, not just one pair.

**Comment by drseb** [comment]
I am responsible for the latter file and I can comment on that. I am not sure about the web-version: @pnrobinson .

The association is not included in that file, as it is marked "susceptibility; modifier" in the source file and these are currently excluded.

**Comment by drseb** [comment]
I hope your question is answered. I will close. If not, please re-open.

**Comment by pnrobinson** [comment]
Can we reopen, https://omim.org/entry/203200 is not a susceptibility, it is an autosomal recessive condition. There might be some bug somewhere. These are the entries in phenotype.hpoa
```
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0000007	OMIM:203200	TAS					I	HPO:skoehler[2009-02-17]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0000505	OMIM:203200	TAS					P	HPO:skoehler[2009-02-17]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0000639	OMIM:203200	TAS					P	HPO:skoehler[2009-02-17]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0002297	OMIM:203200	TAS					P	HPO:skoehler[2010-06-20]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0007603	OMIM:203200	TAS					P	HPO:skoehler[2009-02-17]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0007894	OMIM:203200	TAS			PHPO:skoehler[2012-10-17]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0007750	OMIM:203200	TAS			PHPO:skoehler[2012-10-17]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0000486	OMIM:203200	TAS			PHPO:skoehler[2012-10-17]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0005599	OMIM:203200	TAS					P	HPO:skoehler[2013-02-25]
OMIM:203200	ALBINISM, OCULOCUTANEOUS, TYPE II		HP:0000635	OMIM:203200	TAS					P	HPO:skoehler[2013-02-25]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0001010	OMIM:203200	TAS			PHPO:skoehler[2013-02-25]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0001022	OMIM:203200	TAS			PHPO:skoehler[2015-01-21]
OMIM:203200	#203200 ALBINISM, OCULOCUTANEOUS, TYPE II; OCA2;;OCULOCUTANEOUS ALBINISM, TYPE II;;OCULOCUTANEOUS ALBINISM, TYROSINASE-POSITIVE;;ALBINISM IIALBINISM, BROWN OCULOCUTANEOUS, INCLUDED; BOCA, INCLUDED;;BROWN OCULOCUTANEOUS ALBINISM, INCLUDED		HP:0007663	OMIM:203200	TAS			PHPO:skoehler[2015-07-26]
```

**Comment by drseb** [comment]
@pnrobinson : I agree that the association to gene 4948 is a correct one. But this ticket is about the association to 4157, which is also marked as modifier in https://omim.org/entry/203200 . I think we can close.
